cyclonedx/model/component.py

```diff
@@ -186,14 +186,14 @@ class ComponentEvidence:
     """
 
     def __init__(self, *, licenses: Optional[Iterable[LicenseChoice]] = None,
-                 copyright_: Optional[Iterable[Copyright]] = None) -> None:
-        if not licenses and not copyright_:
+                 copyright: Optional[Iterable[Copyright]] = None) -> None:
```

Comment:
At this point, it would be better to have the variable (not the property) called `copyright_` 
because here in the scope, it would shadow pythons function `copyright()`.
As long as we don't call `copyright()` in this method scope, it is safe. 

who would ever call `copyright()`?! it prints the copyright of python ...

Reply:
As above.